### PR TITLE
[FUSEQE-6793] Remove all alerts before alert check

### PR DESCRIPTION
**Pull request by Stefan365**
//test: `@settings-oauth-validate`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/connections/detail/DetailSteps.java [hunk 1]
```diff
@@ -46,4 +56,21 @@ public void changeConnectionDescription(String connectionDescription) {
...
+        if (alerts.isEmpty()) {
+            return;
+        }
+        do {
+            for (SelenideElement alert : alerts) {
+                alert.$(By.cssSelector("button.close")).shouldBe(Condition.visible).click();
+            }
+            alerts = getCloseableAllerts(alertOption);
+        } while (!alerts.isEmpty());
+    }
```
Comment (mkralik3) [inline ui-tests/src/test/java/io/syndesis/qe/steps/connections/detail/DetailSteps.java]: If I am correct, with the classic `while` you don't need if statement here.
```
while(!alerts.isEmpty()){

}
```

**Comment by mkralik3** [comment]
retest this please

**Comment by mkralik3** [comment]
I have added reproducer, which could be affected this PR, to the running test for this PR.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by Stefan365** [comment]
retest this please

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**